apps/expo/package.json

```diff
@@ -45,13 +45,14 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-native": "0.72.10",
+    "react-native-body-highlighter": "3.0.0-beta.6",
```

Comment:
YOOO you found a library for this? Nice!!

Reply:
See if issue persist